src/FunctionApps/BulkImport/Import/main.py

```diff
@@ -0,0 +1,98 @@
+import logging
+import os
+import zipfile
+
+import requests
+import multiprocessing
+import json
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+
+unzipped_directory = "./FhirResources"
```

Comment:
(non-blocking): We'll have to change the input assumption for this function once it hits the Azure environment. For purposes of orchestration, if the plan is to write a master function that calls the other functions with direct parameter handoff, this is easy to accomplish, but if we're going to go with more of an API route, then this gets trickier. Potentially something to talk through soon (though I guess we'll have to see tickets come out of somewhere first).